src/Middleware/ResponseCaching/src/ResponseCachingExtensions.cs

```diff
@@ -6,8 +6,15 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Extension methods for the ResponseCachingMiddleware middleware.
+    /// </summary>
     public static class ResponseCachingExtensions
     {
+        /// <summary>
+        /// Adds the <see cref="ResponseCachingMiddleware"/> for caching HTTP Responses.
```

Comment:
```suggestion
        /// Adds the <see cref="ResponseCachingMiddleware"/> for caching HTTP responses.
```

nit: Responses shouldn't be capitalized